### PR TITLE
Added a guide to fix iCCp chunks in pngs

### PR DESCRIPTION
**Pull request by J08nY**
Added a guide for Linux/Mac/Windows, hopefully this helps users fix their pngs.
Adds to [5362](https://github.com/godotengine/godot/pull/5362)

### CODE REVIEW

File: tutorials/asset_pipeline/managing_image_files.rst [hunk 1]
```diff
@@ -111,3 +111,34 @@ Atlas for a set of images. It is also possible to ask the exporter to
...
 More information on the :ref:`doc_exporting_images` page.
+
+Fixing PNGs
+-----------
+
+With the upgrade of libpng to 1.6.23, libpng became more strict in terms of
+enforcing iCC profile correctness. This means that it now warns when it comes
+across an image with a non-conforming iCC chunk. 
+
+    ERROR: _png_warn_function: iCCP: known incorrect sRGB profile
```
Comment (vnen) [inline tutorials/asset_pipeline/managing_image_files.rst]: It's actually a `WARNING`, not an `ERROR`.

Comment (J08nY) [inline tutorials/asset_pipeline/managing_image_files.rst]: Should be fixed now.

File: tutorials/asset_pipeline/managing_image_files.rst [hunk 2]
```diff
@@ -111,3 +111,34 @@ Atlas for a set of images. It is also possible to ask the exporter to
 scale all images (or selected groups).
 
 More information on the :ref:`doc_exporting_images` page.
+
+Fixing PNGs
```
Comment (neikeq) [inline tutorials/asset_pipeline/managing_image_files.rst]: IMO the header title is too broad. Something like _"Fixing PNGs iCC profile"_ could be better.

Comment (J08nY) [inline tutorials/asset_pipeline/managing_image_files.rst]: Changed it.

**Pull request merged**